lib/sles4sap/ipaddr2.pm

```diff
@@ -1554,7 +1585,7 @@ sub ipaddr2_refresh_repo {
     $args{bastion_ip} //= ipaddr2_bastion_pubip();
     ipaddr2_ssh_internal(id => $args{id},
         cmd => 'sudo zypper ref',
-        timeout => 240,
+        timeout => 600,
```

Comment:
I increased timeout already to 240, was not enough? Can we say `zypper ref` is slower when using IBSm?

Reply:
During the test,  one VR failed in timeout in 240s, so I increase it to 600s